sources/tests/test.cpp

```diff
@@ -6,7 +6,7 @@ class MockBoard : public Board {
  public:
   int recentId_;
   MockBoard(int w, int h) : Board(w,h) { }
-  virtual void addWorm(WormType type, int x, int y) override {
+  virtual void addWorm(WormType, int x, int y) override {
```

Comment:
I understand that type is not used, but it looks a bit weird to me when some arguments have names and some don't.

Reply:
In this case I can use -Wall -Wextra -Werror flags. It's bad?

Reply:
I looks like you may add in the very first line of the function:
`(void) type;`
To make this error disappear.

https://stackoverflow.com/questions/1486904/how-do-i-best-silence-a-warning-about-unused-variables

But in this case I leave it up to your judgement what is better. Flags should stay there to compile, they are useful.